packages/webcomponents/src/components/business-form-stepped/business-core-info/business-core-info-form-step.tsx

```diff
@@ -23,17 +24,81 @@ import { CoreBusinessInfo, ICoreBusinessInfo } from '../../../api/Business';
   styleUrl: 'business-core-info-form-step.scss',
 })
 export class BusinessCoreInfoFormStep {
-  @Prop() formController: FormController;
+  @Prop() authToken: string;
+  @Prop() businessId: string;
+  @State() formController: FormController;
   @State() errors: any = {};
-  @State() genericInfo: ICoreBusinessInfo = {};
+  @State() coreInfo: ICoreBusinessInfo = {};
+  @Event({ bubbles: true }) submitted: EventEmitter<{ data?: any}>;
+  @Event() formLoading: EventEmitter<boolean>;
+  @Event() serverError: EventEmitter<{ data: any, message: BusinessFormServerErrors }>;
 
   constructor() {
     this.inputHandler = this.inputHandler.bind(this);
+    this.sendData = this.sendData.bind(this);
```

Comment:
Here we could use arrow functions as well

Reply:
Very nice to do this instead of binding stuff in the constructor. This removes a handful of lines from several files. 🙏🏻